mmdet/models/detectors/kd.py

```diff
@@ -0,0 +1,92 @@
+import torch
+
+from ..builder import DETECTORS
+from .single_stage import SingleStageDetector
+
+
+@DETECTORS.register_module()
+class KnowledgeDistillationSingleStageDetector(SingleStageDetector):
+
+    def __init__(self,
+                 backbone,
+                 neck,
+                 bbox_head,
+                 teacher_config='',
+                 teacher_model='',
```

Comment:
Better change this key to `teacher_ckpt`.  It may support `str` or `None` input and set `None` as default. In this way, the users can set the teacher model to train from scratch.